tests/test_serializers.py

```diff
@@ -1,190 +1,221 @@
 import json
 
-from mock import Mock
+from django.db import models
+from mock import Mock, patch
 
-from tests import SerializerTestCase, TestPreference, TestUser
-from madprops.serializers import PropertySerializer
-from rest_framework.serializers import ValidationError
+from madprops.serializers import PropertySerializer, PropertiesOwnerSerializer
```

Comment:
I101 Imported names are in the wrong order. Should be PropertiesOwnerSerializer, PropertySerializer